### DISCUSSION TITLE
Authentication failed installing extension

### DISCUSSION THREAD

**Discussion by camilosoler** [shivammathur/homebrew-extensions#3990] [General]
I'm trying to install an extension with:
brew install shivammathur/extensions/mongodb@7.4
but it asks me for a username and password from GitHub. I forgot to use the tap command previously, so I already did that, but the issue persists.
Can anyone help me?

**Comment by shivammathur** [comment]
@camilosoler I'm not able to reproduce the issue
Test: https://github.com/shivammathur/test-setup-php/actions/runs/9425332704/job/25966876645#step:3:10

Please run `brew update -dv` and then `brew install -v shivammathur/mongodb@7.4` and provide me the logs if you are still getting the request for GitHub credentials.

**Comment by camilosoler** [comment]
It works! 
`brew install -v shivammathur/extensions/mongodb@7.4`
Thanks!
